docarray/array/array.py

```diff
@@ -182,13 +182,37 @@ def __getitem__(self, item):  # note this should handle slices
         else:
             return self.__list_getitem__(item)
 
+    def __get_stacked_document__(self, item):
+        """
+        In stacked mode this return the Document where the filed stacked are put to None
+        In unstacked mode (default) this is the same as __getitem__
+
+        :param item:
+        :return: Document
+        """
+        if self.is_stacked():
+            doc = super().__getitem__(item)
+            for field in self._columns.keys():
+                setattr(doc, field, None)
+            return doc
+        else:
+            return self.__list_getitem__(item)
```

Comment:
What is the difference between this and `super().__getitem__(item)`? Why does line 194 do it the other way?
Could this be simplified?:
```python
doc = super().__getitem__(item)
if self.is_stacked():
    for field in self._columns.keys():
        setattr(doc, field, None)
return doc
```

Reply:
yes it can be